_pages/en/open-source-code-form-validation.html

```diff
@@ -0,0 +1,252 @@
+---
+layout: default
+ref: formValidation
+lang: en
+permalink: /en/open-source-code-form-validation.html
+---
+
+<h1 property="name" id="wb-cont">Form with validation for open source code</h1>
+<div class="wb-frmvld">
+<form action="#" method="get" id="validation">
+
+<legend><span class="field-name"> Version of the metadata schema</span></legend>
+	<div class="form-group">
+		<label for="shemaVersion" class="required"><span class="field-name">schemaVersion</span> <strong class="required">(required)</strong></label>
+			<select class="form-control" id="shemaVersion" name="shemaVersion" required="required">
+				<option></option>
+				<option value="dr">1.0</option>
+			</select>
+	</div>
+        
+<legend><span class="field-name"> Administration code</span></legend>
+	<div class="form-group">
+		<label for="adminCode" class="required"><span class="field-name">adminCode</span> <strong class="required">(required)</strong></label>
+			<select class="form-control" id="adminCode" name="adminCode" required="required">
```

Comment:
Display names of administrations as options.  Use example from codeTable filter. Use adminCode as value.